toolkit/tools/scheduler/schedulerutils/buildlist.go

```diff
@@ -101,8 +102,15 @@ func filterLocalPackagesOnly(packageVersionsInConfig []*pkgjson.PackageVer, inpu
 	return
 }
 
-// ReadReservedFilesList updates the list of reserved files from the manifest file passed in.
+// ReadReservedFilesList updates the list of reserved files (such as toolchain RPMs) from the manifest file passed in.
```

Comment:
>updates [](http://example.com/codeflow?start=25&length=7)

Not part of your PR, but since we're here: the comment is incorrect. We're reading the list of reserved files, not updating it.